parastell/magnet_coils.py

```diff
@@ -250,23 +250,29 @@ def import_step_cubit(self):
 
         self.volume_ids = list(range(first_vol_id, last_vol_id + 1))
 
-    def export_step(self, step_filename="magnet_set", export_dir=""):
-        """Export CAD solids as a STEP file via CadQuery.
+    def export_components(
```

Comment:
Instead of calling this `export_components`, since there's only one component being exported, maybe we should call this something like `export_cad`, and do the same with the in-vessel build for consistency.